### PR TITLE
Fixup predict

### PR DESCRIPTION
**Pull request by NickleDave**

### CODE REVIEW

**Comment by NickleDave** [comment]
@yardencsGitHub and @SamuelBrudner this PR should fix predict.

@SamuelBrudner if you could test the `autoannotate` tutorial again now that I've revised, it would be much appreciated.
I'm not sure if it will show up on the readthedocs page yet, but you should be able to view it rendered directly on GitHub, e.g. here it is in the commit
https://github.com/NickleDave/vak/blob/68bba5865436ec7d705fdca82b4ee021dc86bf81/doc/tutorial/autoannotate.rst

Thank you!

**Comment by NickleDave** [comment]
Went ahead and merged

**Comment by yardencsGitHub** [comment]
Will test predict soon.
In the meantime I ran some train+eval on another canary.. just to see what
happens in larger training sets (here 6000 sec.)
Attached are the eval results on a 5000 sec test set.
Ran once with the max-val-acc checkpoint and once with the end of training
checkpoint.
Very high accuracy results .. I think

I'm still unable to create learning curves starting with small training
sets.
When you're back in business (and I hope the travel and moving goes
smoothly in gator-land :) we should pick that up.

Config:
[PREP]
labelset = "range: 1-30"
spect_format = "mat"
annot_file =
"D:\\YardenBirds\\llb16\\llb16_annotation_May_2019_alexa_4TF.mat"
annot_format = "yarden"
data_dir = "D:\\YardenBirds\\llb16\\annotated"
test_dur = 5000
train_dur = 6000
val_dur = 250
output_dir = "D:\\YardenBirds\\llb16\\train"

[SPECT_PARAMS]
fft_size = 1024
step_size = 64
freq_cutoffs = [ 500, 10000,]
thresh = 6.25

[DATALOADER]
window_size = 370

[EVAL]
csv_path = "D:\\YardenBirds\\llb16\\train\\annotated_prep_200513_114850.csv"
checkpoint_path =
"D:\\YardenBirds\\llb16\\results\\results_200513_124731\\TweetyNet\\checkpoints\\
checkpoint.pt"
output_dir = "D:\\YardenBirds\\llb16\\train"
labelmap_path =
"D:\\YardenBirds\\llb16\\results\\results_200513_124731\\labelmap.json"
models = "TweetyNet"
batch_size = 8
num_workers = 16
device = "cuda"

[TweetyNet]

**Comment by yardencsGitHub** [comment]
Also, I've been looking for some human speech data with annotated phonemes
(time aligned to audio).
I know of TIMIT but access requires paying.
Do you know of anything publicly-available?
